packages/kbn-pm/src/commands/build.ts

```diff
@@ -13,6 +13,11 @@ export const BuildCommand: ICommand = {
   description: 'Runs a build in the Bazel built packages',
   name: 'build',
 
+  reportTiming: {
+    group: 'kbn build',
+    id: 'overall time',
```

Comment:
//nit Lets just call it `total` for simplicity sake

Reply:
Yeah, I can change it. I agree it will be simpler, but it won't be consistent since our current metrics use "overall time". But we can change that.

Reply:
In the context of metrics this is just reporting a number, I think having "time" in the id helps explain what that number means personally.

Reply:
I think consistency wise it's ok, we're not going to be looking at a lot of historical data with this I think

Reply:
I updated them all to be total. @spalger, correct me if I am wrong, but these are separate from the general metrics, and are specifically timings already, right?

Reply:
Yes, my mistake, timings are separate and the value is stored as `ms` so total makes total sense 😉